packages/flutter/test/material/dialog_theme_test.dart

```diff
@@ -51,25 +63,77 @@ RenderParagraph _getTextRenderObject(WidgetTester tester, String text) {
   return tester.element<StatelessElement>(find.text(text)).renderObject! as RenderParagraph;
 }
 
+RenderParagraph _getIconRenderObject(WidgetTester tester, IconData icon) {
+  return tester.renderObject<RenderParagraph>(
+    find.descendant(of: find.byIcon(icon), matching: find.byType(RichText)),
+  );
+}
```

Comment:
```suggestion
RenderParagraph _getIconRenderObject(WidgetTester tester, IconData icon) {
  return tester.renderObject<RenderParagraph>(find.descendant(
    of: find.byIcon(icon),
    matching: find.byType(RichText)
  ));
}
```

This splits `descendant` for better readability.

Reply:
Updated!